cmd/harbor/root/retention/list.go

```diff
@@ -0,0 +1,56 @@
+package retention
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/goharbor/go-client/pkg/sdk/v2.0/client/retention"
+	"github.com/goharbor/harbor-cli/pkg/api"
+	"github.com/goharbor/harbor-cli/pkg/prompt"
+	"github.com/goharbor/harbor-cli/pkg/utils"
+	"github.com/goharbor/harbor-cli/pkg/views/retention/list"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func ListExecutionRetentionCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "list retention execution of the project",
```

Comment:
This is absolute wrong approach here.

As an user I would expect 

```bash
./harbor tag retention list
```
to return me the list of retention rules that I made. Instead of retention executions.

FIX THIS.